### PR TITLE
Added features 1,2 and 3 with proper variable names

### PR DESCRIPTION
**Pull request by sidmojo**

### CODE REVIEW

**Review by nikilagar — CHANGES_REQUESTED** [review]
Use meaningful variable names.

**Comment by nikilagar** [comment]
Give a title to your PR.

File: TicTacToe/src/tictactoe/GameWindow.java [hunk 1]
```diff
@@ -48,13 +48,13 @@ private void endGame(int currentPlayer){
     }
     
     private void check(int currentPlayer){
-        String currentPlayerMove = val1;
+        String currentPlayerMove = pl1symbol;
         if(currentPlayer == 1){
-            currentPlayerMove = val2;
-            jLabel2.setText(name1);
+            currentPlayerMove = pl2symbol;
+            jLabel2.setText(pl1name);
```
Comment (nikilagar) [inline TicTacToe/src/tictactoe/GameWindow.java]: Change the control names too!!! Rest seem fine.

**Review by nikilagar — CHANGES_REQUESTED** [review]

**Comment by sidmojo** [comment]
Control names have been changed.

**Pull request closed without merging**